### PR TITLE
Disable the unique_subject requirement

### PR DESCRIPTION
**Pull request by stefanor**
### Description

This allows more than one key with the same CN to exist in the CA's
database. Which is useful for renewal, as it's the only mode of renewal
supported by easy-rsa.

### Issues Resolved

* Not filed: Unable to create a new client with the same CN as a previous client. E.g. for renewal.

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by stefanor** [comment]
Tests are passing everywhere they do on master.

File: templates/openssl.cnf.erb [hunk 1]
```diff
@@ -25,6 +25,7 @@ default_crl_days= 30
 default_md	= md5
 preserve	= no
 policy		= policy_anything
+unique_subject	= no
```
Comment (JohnRoesler) [inline templates/openssl.cnf.erb]: should this be configurable?

Comment (stefanor) [reply templates/openssl.cnf.erb]: I don't see much point in making it configurable. According to [the docs](https://manpages.debian.org/stretch/openssl/ca.1ssl.en.html#CONFIGURATION_FILE_OPTIONS):

> The default value is yes, to be compatible with older (pre 0.9.8) versions of OpenSSL.

OpenSSL 0.9.8 is old enough to not be a concern, any more, I think. 1.0.0 was released in 2010.

**Comment by stefanor** [comment]
Rebased on master.

**Pull request merged**